packages/server/src/environment.ts

```diff
@@ -58,6 +58,10 @@ const AuthorizationModel = zod.object({
 const HiveModel = zod.union([
   zod.object({
     HIVE_TOKEN: zod.string().optional(),
+    HIVE_GATEWAY_PORT: NumberFromString.optional(),
+    HIVE_SUBGRAPH_PORT: NumberFromString.optional(),
+    HIVE_DEV_REGISTRY_TOKEN: zod.string().optional(),
+    HIVE_STAGING_REGISTRY_TOKEN: zod.string().optional(),
```

Comment:
Make sure all env vars are updated on staging and prod env before merging
And share new dev env vars on slack for us all to align